src/components/modals/WalletListModal.tsx

```diff
@@ -220,59 +220,60 @@ export function WalletListModal(props: Props) {
   // #endregion Renderers
 
   return (
-    <ThemedModal bridge={bridge} onCancel={handleCancel}>
-      <ModalTitle center>{headerTitle}</ModalTitle>
+    <ModalUi4
```

Comment:
We might need to port the `WalletList` to use a `FlatList` from react-native-gesture-handler. When I tried this last week the list was not scrollable.

Reply:
It's scrollable right now. The trick was to let the modal deal with the scroll directly.